index.js

```diff
@@ -40,9 +41,12 @@ function ldnode (argv) {
     // Setting options as local variable
     app.locals.ldp = opts;
 
-    // Session [TODO]
+    // Session
     app.use(session({
-        secret: opts.secret || 'node-ldp',
+        genid: function() {
```

Comment:
This genid is not needed since it is already implemented in here:
https://github.com/expressjs/session/blob/master/index.js#L445-L447